app/server/appsmith-server/src/main/java/com/appsmith/server/filters/PreAuth.java

```
@@ -51,7 +50,7 @@ private Mono<String> getUsername(ServerWebExchange exchange) {
         return exchange.getFormData().flatMap(formData -> {
```

Let's use map here, instead of using `flatMap` and then doing a Mono.just(<some_string>);
Also, do you want to throw an error, if there is no username in the `formData`?